mbq/metrics/contrib/utils.py

```diff
@@ -23,3 +25,9 @@ def get_response_metrics_tags(status_code, path, method):
         'status_code': status_code,
         'status_range': '{}xx'.format(status_code // 100),
     }
+
+
+collector = Collector(
+    namespace="mbq.metrics",
+    tags={"env": _env, "service_name": _service},
```

Comment:
As written, the `env` tag will probably end up being something like `environment.production`